code/src/index.css

```diff
@@ -1,13 +1,185 @@
     monospace;
 }
+
+.thought_wrapper{
+  border: 1px solid #000;
+  box-shadow: 7px 7px #000;
+  width: 50vw;
+  margin: 30px auto;
+  padding: 20px 25px 15px;
+  position: relative;
+}
+.thought_message{
+  word-wrap: break-word;
+}
+.add-thought_input-field {
+  width: calc (50vw 50px);
+  height: 50px;
+  font-family: "Inconsolata", monospace;
+  padding: 10px;
+  margin: 10px 10px 10px 10px;
+  resize: none; 
+}
+
+.characters{
+  font-size:xx-small;
+}
+.add-thought_button{
+  border: none;
+  border-radius: 25px;
+  font-size: 16px;
+  background: #ffadad;
+  padding: 10px 15px;
+  margin: 0 0 15px;
+  margin-top: 10px;
+}
+
+.add-thought_button:hover {
+  cursor: pointer;
+  box-shadow: 0 0 10px rgba(250, 158, 158, 0.6);
+}
+
+.add-thought_button:active {
+  background: rgba(255, 173, 173, 0.8);
+}
+.like-button{
+  border: 1px solid white;
+  color: #f53536;
+  border-radius: 50%;
+  width: 50px;
+  height: 50px;
+  background-color: white;
+  transition: ease-in 0.1s;
+  cursor: pointer;
+  box-sizing: border-box;
+}
+
+.like-button:disabled {
+  border: 2px solid white;
+  background-color: #f53536;
+  color: white;
+}
+
+.btn-border {
+  border-radius: 50%;
+  border: 2px solid #f53536;
+  box-sizing: border-box;
+}
+
+.like-button:hover:not(:disabled) {
+  border: 1px solid white;
+  color: white;
+  background-color: #f53536;
+  transform: scale(1.1);
+}
+
+.likes-info-wrapper {
+  display: flex;
+  flex-direction: row;
+  justify-content: baseline;
+  align-items: center;
+}
+
+.like-button-wrapper {
+  margin-left: -9px;
+}
+
+.spacing-styling {
+  padding-left: 8px;
+}
+
+.like-icon {
+  display: flex;
+  align-self: center;
+  justify-self: center;
+  margin-top: 9px;
+}
+
+.symbols-counter,
+.sub-text-styling {
+  font-size: 12px;
+  margin-top: 0;
+  color: var(--theme-color);
+  margin-bottom: 0;
+}
+.date{
+  font-style: italic;
+}
+.loading-overlay{
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  background: #fff;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.loading-spinner{
+  width: 50px;
+  height: 50px;
+  border:5px solid lightgray;
+  border-radius: 50%;
+  border-left: 5px solid #ffff;
+  animation: spinner infinite 0.5s;
+}
+
+@keyframes spinner {
+  0% {
+    transform: rotate(0);
+  }
+  50% {
+    transform: rotate(180deg);
+  }
+    100%{
+      transform: rotate (360deg);
+    }
+  }
+
+  @media (min-width: 1024px) {
+  .heart-counter__popup__wrapper {
+    right: 24vw;
+    top: 24vh;
+  }
+}
+ @media (max-width: 768px){
+   .thought_wrapper {
+     width: 70vw;
+   }
+   .add-thought_input-field {
+    width: calc(70vw - 50px);
+  }
+  }
+  @media (max-width: 560px) {
+  .thought__wrapper {
+    width: 90vw;
+  }
+
+  .add-thought__input-field {
+    width: calc(90vw - 50px);
+  }
+
+  .heart-counter__popup__wrapper {
+    top: 24vh;
+    right: 11vw;
+  }
+}
+@media (max-width: 375px) {
```

Comment:
What does this .heart-counter__popup__wrapper do? I tried to find it in the code (maybe I'm just staring blind :)) For mobile view you could put it in before the @media to make it mobile first.